gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/MySqlDagManagementStateStoreTest.java

```diff
@@ -96,13 +95,19 @@ public static <T> boolean compareLists(List<T> list1, List<T> list2) {
   }
 
   @Test
-  public void testcanAcquireLeaseOnEntity() throws Exception{
-    Mockito.when(leaseArbiter.isLeaseAcquirable(Mockito.any(DagActionStore.LeaseParams.class))).thenReturn(true);
+  public void testExistsCurrentlyLaunchingSimilarFlowGivesTrue() throws Exception{
+    Mockito.when(leaseArbiter.existsSimilarLeaseWithinConsolidationPeriod(Mockito.any(DagActionStore.LeaseParams.class))).thenReturn(true);
     String flowName = "testFlow";
     String flowGroup = "testGroup";
-    DagActionStore.DagAction dagAction = new DagActionStore.DagAction(flowName, flowGroup, System.currentTimeMillis(), "testJob", DagActionStore.DagActionType.LAUNCH);
-    DagActionStore.LeaseParams leaseParams = new DagActionStore.LeaseParams(dagAction);
-    Assert.assertTrue(dagManagementStateStore.isLeaseAcquirable(leaseParams));
+    Assert.assertTrue(dagManagementStateStore.existsCurrentlyLaunchingSimilarFlow(flowGroup, flowName, any(Long.class)));
```

Comment:
apologies, but I'm not actually familiar w/ what `any(Long.class)` means in a context like this.  I'm familiar w/ such arg matchers setting-up mocking and also to verify prior invocations of a mock - but not within an actual invocation of a non-mock.

guessing: does it choose a random long and pass that as the arg?  it might be better to just pass an actual value, such as `System.currentTimeMillis()`

Reply:
updated